OrbitGl/TimeGraph.cpp

```diff
@@ -167,7 +167,7 @@ void TimeGraph::ZoomTime(float a_ZoomValue, double a_MouseRatio) {
   m_MouseRatio = a_MouseRatio;
 
   static double incrementRatio = 0.1;
-  double scale = a_ZoomValue > 0 ? 1 + incrementRatio : 1 - incrementRatio;
+  double scale = a_ZoomValue > 0 ? 1 + incrementRatio : 1 / (1 + incrementRatio);
```

Comment:
When you're changing this anyway, could you add parenthesis to the arguments of the ternary operator for all the folks who don't know the operator precedence list by heart. :see_no_evil:

Reply:
This is a great suggestion and IMO should just be the default behavior: always add parenthesis to indicate precedence. Harder to make a mistake, easier to read.

Reply:
Thanks! I got used to them, but you are right :)